sdks/csharp/.gitignore

```diff
@@ -0,0 +1,353 @@
+## Ignore Visual Studio temporary files, build results, and

```

Comment:
There is a [.gitignore file](https://github.com/googleforgames/agones/blob/master/.gitignore) already.

Reply:
Does this work? 3237f6924060144f81534c6c818f09a8b21eaf36